pkg/webui/locales/en.json

```diff
@@ -466,8 +466,8 @@
   "console.containers.application-payload-formatters.uplink.title": "Default uplink payload formatter",
   "console.containers.application-payload-formatters.uplink.infoText": "You can use the \"Payload formatter\" tab of individual end devices to test uplink payload formatters and to define individual payload formatter settings per end device.",
   "console.containers.application-payload-formatters.uplink.uplinkResetWarning": "You do not have sufficient rights to view the current uplink payload formatter. Only overwriting is allowed.",
-  "console.containers.application-title-section.application-title-section.lastSeenAvailableTooltip": "The elapsed time since the network registered activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.",
-  "console.containers.application-title-section.application-title-section.noActivityTooltip": "The network has not recently registered any activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.",
+  "console.containers.application-title-section.index.lastSeenAvailableTooltip": "The elapsed time since the network registered activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.",
+  "console.containers.application-title-section.index.noActivityTooltip": "The network has not recently registered any activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.",
```

Comment:
When these rewrites happen, make sure the Japanese messages are also rewritten, in case they exist. Otherwise we loose the Japanese translation.

Reply:
In this case they seem to not exist. But I'll make a note to check